rewrite-weblogic/docs/recipes/migrate-to-jakarta-EE-9_1.md

```diff
@@ -83,6 +86,9 @@ dependencies {
 ```
 2. Run `gradle rewriteRun` to run the recipe.
 
+> [!TIP]  
+> If you just want to dry run the recipe without changing the code, use `gradle rewrite:DryRun`.

```

Comment:
Same: The correct command is `gradle rewriteDryRun`. Reference: https://docs.openrewrite.org/reference/gradle-plugin-configuration#the-dryrun-task